src/stp/new_skills/Rotate.cpp

```diff
@@ -19,7 +19,7 @@ Status Rotate::onUpdate(const StpInfo &info) noexcept {
     int targetDribblerSpeed = targetDribblerPercentage / 100.0 * Constants::MAX_DRIBBLER_CMD();
 
     // Set angle command
-    command.set_w(targetAngle.getAngle());
+    command.set_w(targetAngle);
```

Comment:
```suggestion
    command.set_w(targetAngle.getAngle());
```
targetAngle is of Angle class now